content/en/docs/File Format/implementationstatus.md

```diff
@@ -13,94 +13,96 @@ implementations.
 The value in each box means:
 * ✅: supported
 * ❌: not supported
+* (R/W): partial reader/writer only support
 * (blank) no data
 
 Implementations:
 * `C++`: [parquet-cpp](https://github.com/apache/arrow/tree/main/cpp/src/parquet)
 * `Java`: [parquet-java](https://github.com/apache/parquet-java)
 * `Go`: [parquet-go](https://github.com/apache/arrow-go/tree/main/parquet)
 * `Rust`: [parquet-rs](https://github.com/apache/arrow-rs/blob/main/parquet/README.md)
+* `cuDF`: [cudf](https://github.com/rapidsai/cudf)
 
 
 
 ### Physical types
 
-| Data type                                 | C++   | Java   | Go    | Rust  |
```

Comment:
Simply removed one space in the `Java` column so all cols have a consistent width for aesthetic purposes.